### DISCUSSION TITLE
Create new item in the database

### DISCUSSION THREAD

**Discussion by mystery123sk** [notion-dotnet/notion-sdk-net#406] [Q&A]
I'm sorry if I'm being stupid, but is it possible to add a new item to some database using your API? Do you have some simple example?

**Comment by KoditkarVedant** [comment]
@mystery123sk you may check the integration tests `https://github.com/notion-dotnet/notion-sdk-net/blob/main/Test/Notion.IntegrationTests/DatabasesClientTests.cs` hope that would give some idea.
